diff --git a/src/nhflow_rans_ini.cpp b/src/nhflow_rans_ini.cpp
index b54b5b315..dfd811279 100644
--- a/src/nhflow_rans_ini.cpp
+++ b/src/nhflow_rans_ini.cpp
@@ -134,7 +134,7 @@ void nhflow_rans_io::inflow(lexer* p, fdm_nhf *d, ghostcell* pgc)
     kinval = kinbed * (1.0 - 0.5*(beddist/(d->WL(i,j)>0.0?d->WL(i,j):1.0e20)));
     
     if(p->A560==1 || p->A560==21)
-    epsval = p->cmu*kinval*kinval/(evval>1.0e-20?evval:1.0e20);  
+    epsval = p->cmu*kinval*kinval/(evval>1.0e-20?evval:1.0e20);
     
     else
     epsval = kinval/(evval>1.0e-20?evval:1.0e20);
